### DISCUSSION TITLE
Speed up's with python 3.11

### DISCUSSION THREAD

**Discussion by ThomasMGeo** [kinverarity1/lasio#535] [Ideas]
Know there was some speed testing previously, wonder how much python 3.11 improves this result? Probably minimal, be interesting!

**Comment by dcslagel** [comment]
The speed is up just a little bit between 3.9 and 3.10.  I haven't tried 3.11 yet.

test_read_v12_sample_big  processing times:
3.9 Median: 138.4306
3.10 Median: 124.7611

**Comment by dcslagel** [comment]
Okay, I finally got around to trying Python 3.11.  The median processing time is a little faster but it isn't a wildly improved speed.
Hmm, maybe it is roughly 20% faster than the 3.9 Median.

test_read_v12_sample_big processing times:
3.9 Median: 138.4306
3.10 Median: 124.7611
**3.11 Median: 108.4755**

**Reply by ThomasMGeo** [reply]
Thanks for doing these tests!
